app/views/users/show.html.erb

```diff
@@ -28,7 +30,7 @@
 
 
 
-
+ 
 
 
 
```

Comment:
**[OPTIONA BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you fix the controller specs because i saw that some of them are failing 😨 Kindly fix them to make your controllers tested well 👍